src/parallel/KDDecomposition.cpp

```diff
@@ -191,9 +191,20 @@ void KDDecomposition::readXML(XMLfileUnits& xmlconfig) {
 	global_log->info() << "Generate new vectorization tuner files: " << (_generateNewFiles?"yes":"no") << endl;
 	xmlconfig.getNodeValue("useExistingFiles", _useExistingFiles);
 	global_log->info() << "Use existing vectorization tuner files (if available)?: " << (_useExistingFiles?"yes":"no") << endl;
+	xmlconfig.getNodeValue("vecTunerAllowMPIReduce", _vecTunerAllowMPIReduce);
+	global_log->info() << "Allow an MPI Reduce for the vectorization tuner?: " << (_vecTunerAllowMPIReduce?"yes":"no") << endl;
```

Comment:
```suggestion
	global_log->info() << "Allow an MPI Reduce for the vectorization tuner?: " << std::boolalpha << _vecTunerAllowMPIReduce  << endl;
```
Maybe less error-prone alternative but if applied should be done everywhere...

Reply:
yeah, not in this PR :P